packages/docs/docs/concepts/progressive-loading/stackProgressive.md

```diff
@@ -0,0 +1,109 @@
+---
+id: stackProgressive
+---
+
+# Stack Progressive Loading
+
+Images for the stack viewport can be loaded with a lower resolution/lossy
+version first, followed by increasingly higher resolutions, and finally
+the final version being a lossless representation.
+
+For HTJ2K, this is done automatically when the image is encoded in progressive
+resolution order by using a streaming reader that returns lower resolution versions
+of the image as they are available.
+
+For other image types, a separate lower resolution/lossy version is required.
+The Static DICOMweb toolkit includes some options to create such images.
+
+# Performance
+
+In general, about 1/16-1/10th of the image is retrieved for the lossy/first
+version of the image. This results in about a 10x speed improvement to first
+images. It is affected fairly strongly by overall image size, network performance
+and compression ratios.
+
+The full size images are 3036 x 3036, while the JLS reduced images are 759 x 759
```

Comment:
It can be rephrased as: "For instance, you can observe the results of our benchmark, which demonstrate various scenarios. In the table below, the full-size images are displayed..."